theoplayer/getting-started/01-sdks/01-web/01-getting-started-accessibility.mdx

```diff
@@ -0,0 +1,63 @@
+
+There are three levels to the WCAG: A, AA, and AAA:
+
+- A: the basic/bare minimum accessibility functionalities
+- AA: decently accessible, where most organisations tend to stand
+- AAA: very accessible, top tier accessibility
+
+What is new in WCAG?
+WCAG 2.1 was updated in 2018. The success criteria from WCAG 2.0 are included verbatim in WCAG 2.1, but 17 additions were included in 2.1 to address mobile accessibility, viewers with low vision and viewers with cognitive and learning disabilities. WCAG 2.0 and 2.1 are backwards compatible, or in other words, if an organisation adheres to 2.1 they also adhere to 2.0.
+
+The additions in 2.1 require online content to be adaptable, distinguishable, compatible, keyboard accessible, have functional modalities, be safe for those with seizures or physical reactions, and have ‘enough time’ capabilities. To view the technical documentation of the WCAG 2.1 and use case examples, [visit their website here](https://www.w3.org/WAI/standards-guidelines/).
+
+WCAG 2.2 was updated in 2024. The 2.0 and 2.1 success criteria are essentially the same in 2.2, with one exception: Parsing is obsolete and removed from WCAG 2.2.
+
+OptiView Player is a single player across all major mobile devices, platforms and browsers, and our video player solution is upgraded with accessibility capabilities to adhere to WCAG across all of them, out of the box.
+
+## User Agent Accessibility Guidelines (UAAG)
+
+Under the WCAG is the User Agent Accessibility Guidelines (UAAG), which encompasses web browsers, browser extensions and media players, and the UAAG targets developers. The UAAG was created to primarily provide alternative information for those who want to remain compliant with WCAG, but are using a variety of different technologies and platforms. For example, some accessibility needs are met through the browser the viewer is using, while others are better met through the media content.
+
+UAAG is seen as relevant for the future of accessibility, especially for mobile device accessibility. Mobile devices show the need for accessibility within a combination of UI, the content and applications used. UAAG also highlights the need for more accessibility features that work seamlessly across devices, platforms, and browsers.
+If you would like to explore more about UAAG specifically, check out the documentation with [standards and guidelines from WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/).
+
+## Why is WCAG important for Video
+
+Complying with the WCAG is crucial for all publishers of online content, and particularly in video content. By ignoring accessibility, you are losing a large chunk of your audience. By a large chunk, we mean 15% of the world’s population who have at least one disability. By implementing accessibility features you are including everyone, reaching more viewers and increasing your traffic and SEO rankings.
```

Comment:
nitpick: @dcoffey3296 I'd change it to:
"Complying with the WCAG is crucial for all publishers of online content, particularly video content."